features/Dockefile.buildimage

```diff
@@ -0,0 +1,25 @@
+FROM fedora:30 as build-tools
+ENV GOPATH /go
+ENV GOBIN /go/bin
+ENV GOCACHE /go/.cache
+ENV PATH=$PATH:$GOPATH/bin
+
+# rpms required for building and running test suites
+RUN dnf -y install \
+    golang-1.12.13-1.fc30 \
```

Comment:
is this golang specific version needed?
If so, what about having an environment variable instead?

Reply:
Nope, but on the long term we want to use the same Dockerfile for local builds.
By pinning a specific version we make sure it won't change over time and we get reproducible builds.

Ok for the env variable.

Reply:
Done